generators/angular/templates/src/main/webapp/app/home/home.component.html.ejs

```diff
@@ -26,25 +26,27 @@
 
     <p class="lead" <%= jhiPrefix %>Translate="home.subtitle">__jhiTransformTranslate__('home.subtitle')</p>
 
-    <div [ngSwitch]="account !== null">
-      <div class="alert alert-success" *ngSwitchCase="true">
-      @if (account) {
-        <span id="home-logged-message" <%= jhiPrefix %>Translate="home.logged.message"
-          [translateValues]="{ username: account.login }">__jhiTransformTranslate__('home.logged.message', { "username": "{{ account.login }}" })</span>
-      }
-      </div>
-
-      <div class="alert alert-warning" *ngSwitchCase="false">
-        <span <%= jhiPrefix %>Translate="global.messages.info.authenticated.prefix">__jhiTransformTranslate__('global.messages.info.authenticated.prefix')</span>
-        <a class="alert-link" (click)="login()" <%= jhiPrefix %>Translate="global.messages.info.authenticated.link">__jhiTransformTranslate__('global.messages.info.authenticated.link')</a><span <%= jhiPrefix %>Translate="global.messages.info.authenticated.suffix">__jhiTransformTranslate__('global.messages.info.authenticated.suffix')</span>
-      </div>
-      <%_ if (generateUserManagement) { _%>
+    <div>
+      @if (account !== null) {
```

Comment:
I tried to keep the same HTML as before. In this case, I think the `if` should be before the `<div>`.

Reply:
The original code is a ngSwitch, the div is written for true or false.
This change would be organizational.

Reply:
with the `ngSwitch` this `div` was always present